rest_framework_simplify/mapper.py

```diff
@@ -70,6 +70,8 @@ def dict_camelcase_to_underscore(obj):
             return new_dict
         elif isinstance(obj, list):
             new_list = []
+            if all(isinstance(item, int) or isinstance(item, str) for item in obj):
+                return obj
```

Comment:
A better way to do it might be to inspect each one on an individual basis, which would make it so it'd still work if you had an array like this:

ary = [
1,
'two',
{ 'id': 'three'}
}

Reply:
I like this change, makes it a bit more robust

Reply:
> I like this change, makes it a bit more robust

k it's ready i think.